tests/phpunit/tests/comment/metaCache.php

```diff
@@ -108,6 +108,8 @@ public function test_comment_meta_should_be_lazy_loaded_for_all_comments_in_comm
 
 		$this->go_to( get_permalink( $p ) );
 
+		$this->assertTrue( have_posts() );
+
 		if ( have_posts() ) {
```

Comment:
The `if ( have_posts() ) {` can be removed as it will always be true if the `assertTrue()` didn't fail.